### PR TITLE
Airbyte CDK: fix file-based deps

### PR DESCRIPTION
**Pull request by artem1205**
## What

fix deps for file based CDK
Reason: 
pdf2image and "pdfminer.six" are missing

## How

move packages to [tool.poetry.dependencies]

### CODE REVIEW

**Review by natikgadzhi — COMMENTED** [review]
I think previously the unit tests were still passing, right?

Can you please add the tests that would use these dependencies, so we don't end up in this situation again?

**Comment by artem1205** [comment]
@natikgadzhi ,

> I think previously the unit tests were still passing, right?

Airbyte CDK tests use command in CI : `poetry install --all-extras`, that is why we don't see any problems with deps.

> Can you please add the tests that would use these dependencies, so we don't end up in this situation again?

We already have CAT/ integration tests in connectors that use CDK deps, see  https://github.com/airbytehq/airbyte/actions/runs/8480072489/job/23235149671 as example

**Review by erohmensing — APPROVED** [review]
Thank you for tracking this down!

File: airbyte-cdk/python/pyproject.toml [hunk 1]
```diff
@@ -62,8 +64,6 @@ datamodel_code_generator = "0.11.19"
 freezegun = "*"
 mypy = "*"
 pandas = "2.0.3"
```
[the remark this answers is not in the document]
Comment (artem1205) [reply airbyte-cdk/python/pyproject.toml:66]: @erohmensing , I'm ok with this. The only line with pandas I found is:
https://github.com/airbytehq/airbyte/blob/eb31e4d2ba796f893e1740b5e7a298fd346400ef/airbyte-cdk/python/unit_tests/sources/file_based/in_memory_files_source.py#L16

**Pull request merged**